app/views/internal/configs/show.html.erb

```diff
@@ -105,6 +97,32 @@
           </div>
         </div>
 
+        <div class="card mt-3">
```

Comment:
Split out the form sections, one for social media and one sections holds the emails.